UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -138,10 +168,60 @@ public void LazyInitialization()
             }
         }
 
+        /// <summary>
+        /// Enable stepping of the Academy during the FixedUpdate phase.  This is done by creating a temporary
+        /// GameObject with a MonoBehavior that calls Academy.EnvironmentStep().
+        /// </summary>
+        public void EnableAutomaticStepping()
+        {
+            if (m_FixedUpdateStepper != null)
+            {
+                return;
+            }
+
+            m_StepperObject = new GameObject("AcademyFixedUpdateStepper");
+            // Don't show this object in the hierarchy
+            m_StepperObject.hideFlags = HideFlags.HideInHierarchy;
+            m_StepperObject.AddComponent<AcademyFixedUpdateStepper>();
```

Comment:
AddComponent returns the added component.
```suggestion
            m_FixedUpdateStepper  = m_StepperObject.AddComponent<AcademyFixedUpdateStepper>();
```